src/main/kotlin/com/spectralogic/rioclient/RioClient.kt

```diff
@@ -39,16 +39,15 @@ class RioClient(
     private val username: String = "spectra",
     private val password: String = "spectra",
     private val requestTimeout: Long = 60L * 1000L, // 60 seconds
-    longLivedToken: String? = null,
-    reauthorizeDeltaSeconds: Long = 3_600L // TODO: ESCP-3450
+    private val longLivedToken: String? = null
```

Comment:
Any reason to turn this into a `val`? I don't think we need to hold onto it after construction?

Reply:
I implemented:
    suspend fun getLongLivedToken(): String? = longLivedToken
so that the shutdown hook of RioCruise could try and delete it.

Reply:
I guess it would be better to RC just to keep it's own copy.